metadata-ingestion/tests/integration/mlflow/test_mlflow_source.py

```diff
@@ -1,7 +1,9 @@
+import uuid
 from pathlib import Path
 from typing import Any, Dict, TypeVar
 
 import pytest
+from _pytest.monkeypatch import MonkeyPatch
```

Comment:
can we import this via `pytest.MonkeyPatch`?

Reply:
done